.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        ruby-version: [2.3, 2.4, 2.5, 2.6, 2.7]
+        ruby-version: [2.3, 2.4, 2.5, 2.6, 2.7, '3.0']
```

Comment:
I guess we should probably be wrapping all of these in quotes...

Reply:
Indeed.